spec/features/saml/loa1_sso_spec.rb

```diff
@@ -120,7 +120,6 @@
       saml_authn_request = auth_request.create(saml_settings)
       visit saml_authn_request
       register_user
-      click_on t('users.personal_key.get_another')
```

Comment:
Since we are no longer allowing users to generate a new key during sign up, this entire scenario should be removed.